software/webapp/src/main/java/org/apache/brooklyn/entity/proxy/AbstractNonProvisionedControllerImpl.java

```diff
@@ -57,19 +57,19 @@
     public AbstractNonProvisionedControllerImpl() {
         this(MutableMap.of(), null, null);
     }
-    public AbstractNonProvisionedControllerImpl(Map properties) {
+    public AbstractNonProvisionedControllerImpl(Map<?,?> properties) {
```

Comment:
In https://github.com/apache/brooklyn-library/pull/10 (not merged or reviewed yet by anyone), I juste deleted these constructors.